src/main/java/spring/impls/SimpleUser.java

```diff
@@ -0,0 +1,32 @@
+package spring.impls;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+@Log4j
+@Getter
+@Setter
+@EqualsAndHashCode
+public class SimpleUser implements Lyricist {
```

Comment:
It looks the same to class Administrator. Why not use inheritance or something like this?
Let's not break [DRY](https://ru.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself) principle.